tools/common.sh

```diff
@@ -37,6 +37,18 @@ for python in \
     fi
 done
 
+for python3 in \
+    "$PYTHON3" \
```

Comment:
in general, I don't know if we care to use curly brackets for variables (like ${var}) consistently or not

Reply:
It works like this; this code is basically the same some code above which sets the $PYTHON variable for Python 2.